README.md

```diff
@@ -24,19 +24,22 @@ Cost Monitoring Construct provides the following features:
 - **Budgets:** Allows you to set budgets for each applications. It will automatically set up alerts to notify you when your actual costs exceed your budgeted costs. It also continues to track the cost and sending alert if an application continues to cost drastically.
 - **Integration:** Integrates with various tools and monitoring services, such as AWS Cost Explorer and Datadog.
 
-# What is it useful for?
+> [!WARNING]
+> Tags used to create and monitor budgets must first be [activated as cost allocation tags](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/activating-tags.html). Ensure your chosen one is activated, otherwise the budgets won't work!
```

Comment:
@wtfzambo  maybe also would be good to mention that the default tag key is `cm:application` so users who are leaving it to default know which tag key they should look for?